src/backtracking.py

```diff
@@ -1,32 +1,30 @@
-from AI_Sudoku.sudoku import Sudoku
-from AI_Sudoku.field import Field
+from src.field import Field
+from src.sudoku import Sudoku
 
 
 # noinspection PyShadowingBuiltins
-class ForwardChecking(Sudoku):
+class Backtracking(Sudoku):
```

Comment:
Why did you change the files? forward_checking.py -> backtracking.py and vice versa?